packages/design-tokens/css/default-theme.css

```diff
@@ -1,302 +1,1706 @@
+  --typography-paragraph-small-font-size-id: --typography-paragraph-small-font-size;
+  --typography-paragraph-small-line-height: 1.125rem;
+  --typography-paragraph-small-line-height-id: --typography-paragraph-small-line-height;
+  --typography-paragraph-small-letter-spacing: normal;
+  --typography-paragraph-small-letter-spacing-id: --typography-paragraph-small-letter-spacing;
+  --typography-paragraph-extra-small-font-family: "Inter", "Noto Sans",
+    Helvetica, Arial, sans-serif;
+  --typography-paragraph-extra-small-font-family-id: --typography-paragraph-extra-small-font-family;
+  --typography-paragraph-extra-small-font-weight: 400;
+  --typography-paragraph-extra-small-font-weight-id: --typography-paragraph-extra-small-font-weight;
+  --typography-paragraph-extra-small-font-size: 0.75rem;
+  --typography-paragraph-extra-small-font-size-id: --typography-paragraph-extra-small-font-size;
+  --typography-paragraph-extra-small-line-height: 1.125rem;
+  --typography-paragraph-extra-small-line-height-id: --typography-paragraph-extra-small-line-height;
+  --typography-paragraph-extra-small-letter-spacing: normal;
+  --typography-paragraph-extra-small-letter-spacing-id: --typography-paragraph-extra-small-letter-spacing;
+  --typography-paragraph-bold-font-weight: 600;
+  --typography-paragraph-bold-font-weight-id: --typography-paragraph-bold-font-weight;
+  --typography-button-primary-font-family: "Inter", "Noto Sans", Helvetica,
+    Arial, sans-serif;
+  --typography-button-primary-font-family-id: --typography-button-primary-font-family;
+  --typography-button-primary-font-weight: 700;
+  --typography-button-primary-font-weight-id: --typography-button-primary-font-weight;
+  --typography-button-primary-font-size: 1.125rem;
+  --typography-button-primary-font-size-id: --typography-button-primary-font-size;
+  --typography-button-primary-line-height: 1.5rem;
+  --typography-button-primary-line-height-id: --typography-button-primary-line-height;
+  --typography-button-primary-letter-spacing: normal;
+  --typography-button-primary-letter-spacing-id: --typography-button-primary-letter-spacing;
+  --typography-button-secondary-font-family: "Inter", "Noto Sans", Helvetica,
+    Arial, sans-serif;
+  --typography-button-secondary-font-family-id: --typography-button-secondary-font-family;
+  --typography-button-secondary-font-weight: 500;
+  --typography-button-secondary-font-weight-id: --typography-button-secondary-font-weight;
+  --typography-button-secondary-font-size: 1rem;
+  --typography-button-secondary-font-size-id: --typography-button-secondary-font-size;
+  --typography-button-secondary-line-height: 1.5rem;
+  --typography-button-secondary-line-height-id: --typography-button-secondary-line-height;
+  --typography-button-secondary-letter-spacing: normal;
+  --typography-button-secondary-letter-spacing-id: --typography-button-secondary-letter-spacing;
   --kz-var-theme-key: zen;
+  --kz-var-theme-key-id: --kz-var-theme-key;
```

Comment:
On one hand, I don't think we need this. On the other, it certainly won't hurt to add it in. 
Did you have a use case in mind for when someone might use `--kz-var-theme-key-id:`

Reply:
well, these files are auto generated.
To remove this variable would mean to add an exception in the generator function which i'm not sure about.
Actually, does anyone use these CSS files anyway?

Reply:
>Actually, does anyone use these CSS files anyway? 
I'm a bit doubtful 😅 
If it's just this variable that is a bit of an anomaly this is probably fine.